curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/675788f025208de23bedb6bc.md

```diff
@@ -0,0 +1,1569 @@
+---
+id: 675788f025208de23bedb6bc
+title: Step 95
+challengeType: 0
+dashedName: step-95
+---
+
+# --description--
+
+We have identified a bug in the music player with handling an empty playlist. When all songs in the playlist are deleted, a "Reset Playlist" button correctly appears, allowing the user to restore the playlist. However, if the shuffle button is clicked while the playlist is empty, the "Reset Playlist" button disappears. This issue arises because the shuffle() function does not account for the edge case of an empty playlist. To fix this, update the shuffle() function to handle this scenario appropriately. Ensure that the "Reset Playlist" button remains visible and functional even when the playlist is empty and the shuffle button is clicked.
```

Comment:
this text doesn't follow the style guide for step challenges
 - never use first person ("we", "I", "let's")
 - you need to use backticks to surround code things

Also maybe it needs to be divided into two paragraphs, it's quite long.